Source/WebCore/platform/graphics/win/MediaPlayerPrivateMediaFoundation.cpp

```diff
@@ -2796,53 +2795,6 @@ HRESULT MediaPlayerPrivateMediaFoundation::Direct3DPresenter::presentSample(IMFS
     return hr;
 }
 
-void MediaPlayerPrivateMediaFoundation::Direct3DPresenter::paintCurrentFrame(WebCore::GraphicsContext& context, const WebCore::FloatRect& destRect)
```

Comment:
We shouldn't move method. Using #if USE(CAIRO) and USE(SKIA) is better.

Reply:
Are you planning to add MediaPlayerPrivateMediaFoundationSkia.cpp? After finish skia migration, will you remove the MediaPlayerPrivateMediaFoundationSkia.cpp?